### PR TITLE
[INFINITY-1875] Only use foldered service names on DC/OS 1.10+.

### PR DESCRIPTION
**Pull request by jsrodman**

### CODE REVIEW

File: testing/sdk_utils.py [hunk 1]
```diff
@@ -44,4 +44,12 @@ def try_throws_n_times(predicate, attempts=1, sleep_seconds=0):
...
 
-        time.sleep(sleep_seconds)
\ No newline at end of file
+        time.sleep(sleep_seconds)
+
+def get_foldered_name(service_name):
+    # DCOS 1.9 & earlier don't support "foldered", service names aka marathon
+    # group names
+    if shakedown.dcos_version_less_than("1.10"):
+        return "test__integration__" + service_name
```
Comment (nickbp) [inline testing/sdk_utils.py]: double underscores in service names should fail* regardless of dcos version. should probably just use single underscores

* https://github.com/mesosphere/dcos-commons/blob/master/sdk/scheduler/src/main/java/com/mesosphere/sdk/config/validate/ServiceNameCannotContainDoubleUnderscores.java

**Comment by jsrodman** [comment]
retest this please

**Comment by jsrodman** [comment]
Ironically, we won't really know if this helps in a categorical way until the nightly happens, though I did test locally with template and it passed just fine.

**Comment by benclarkwood** [comment]
retest this please

**Comment by jsrodman** [comment]
retest this please

**Comment by jsrodman** [comment]
"Fixups" got a success, i guess, and the failures post merging in master are entirely unrelated (probably uninstall troubles.)  Merging.

**Pull request merged**